pallets/identity/rpc/runtime-api/src/lib.rs

```diff
@@ -34,6 +34,13 @@ pub enum LinkType {
     AssetOwnership,
     NoData,
 }
+#[derive(Encode, Decode, PartialEq, Eq)]
+#[cfg_attr(feature = "std", derive(Debug, Serialize, Deserialize))]
+pub enum DidStatus {
+    Unknown,
+    Valid,
```

Comment:
I think it will be better to use something like `Exists`/`Present`/`Registered` instead `Valid` for clarity.